v3/1upkeyboards/pi60_rgb_v2/pi60_rgb_v2.json

```
@@ -0,0 +1,243 @@
+                  "Gradient Left/Right",
+                  "Breathing",
+                  "Band Sat.",
+                  "Band Val.",
+                  "Pinwheel Sat.",
+                  "Pinwheel Val.",
+                  "Spiral Sat.",
+                  "Spiral Val.",
+                  "Cycle All",
+                  "Cycle Left/Right",
+                  "Cycle Up/Down",
+                  "Rainbow Moving Chevron",
+                  "Cycle Out/In",
+                  "Cycle Out/In Dual",
+                  "Cycle Pinwheel",
+                  "Cycle Spiral",
+                  "Dual Beacon",
+                  "Rainbow Beacon",
+                  "Rainbow Pinwheels",
+                  "Raindrops",
+                  "Jellybean Raindrops",
+                  "Hue Breathing",
+                  "Hue Pendulum",
+                  "Hue Wave",
+                  "Pixel Rain",
+                  "Pixel Flow",
+                  "Pixel Fractal",
+                  "Typing Heatmap",
+                  "Digital Rain",
+                  "Solid Reactive Simple",
+                  "Solid Reactive",
+                  "Solid Reactive Wide",
+                  "Solid Reactive Multiwide",
+                  "Solid Reactive Cross",
+                  "Solid Reactive Multicross",
+                  "Solid Reactive Nexus",
+                  "Solid Reactive Multinexus",
+                  "Splash",
+                  "Multisplash",
+                  "Solid Splash",
+                  "Solid Multisplash"
+                ]
+              },
+              {
+                "showIf": "{id_qmk_rgb_matrix_effect} != 0 && {id_qmk_rgb_matrix_effect} != 1 && {id_qmk_rgb_matrix_effect} != 2 && {id_qmk_rgb_matrix_effect} != 3 && {id_qmk_rgb_matrix_effect} != 4",
+                "label": "Effect Speed",
+                "type": "range",
+                "options": [0, 255],
+                "content": ["id_qmk_rgb_matrix_effect_speed", 3, 3]
+              },
+              {
+                "showIf": "{id_qmk_rgb_matrix_effect} == 2",
+                "label": "Mod Hue",
+                "type": "range",
+                "options": [0, 255],
+                "content": ["id_qmk_rgb_matrix_effect_speed", 3, 3]
+              },
+              {
+                "showIf": "{id_qmk_rgb_matrix_effect} == 3 || {id_qmk_rgb_matrix_effect} == 4",
+                "label": "Gradient Range",
+                "type": "range",
+                "options": [0, 255],
+                "content": ["id_qmk_rgb_matrix_effect_speed", 3, 3]
+              },
+              {
+                "showIf": "{id_qmk_rgb_matrix_effect} != 0 && {id_qmk_rgb_matrix_effect} != 23 && {id_qmk_rgb_matrix_effect} != 27 && {id_qmk_rgb_matrix_effect} != 28",
+                "label": "Color",
+                "type": "color",
+                "content": ["id_qmk_rgb_matrix_color", 3, 4]
+              }
+            ]
+          }
+        ]
+      }
+    ],
```

```suggestion
    "menus": [ "qmk_rgb_matrix" ],
```
Could be simplified to this?